upgrading/upgrade-operator.adoc

```diff
@@ -21,4 +26,5 @@ include::modules/rollback-operator-upgrades-console.adoc[leveloffset=+1]
 
 * xref:../installing/installing_ocp/install-central-ocp.adoc#install-central-operator_install-central-ocp[Installing Central using the Operator method]
 * link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/understanding-operators#olm-workflow[Operator Lifecycle Manager workflow]
-* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update]
\ No newline at end of file
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update]
+* add link to Upgrading from {product-title-short} 3.74 and earlier to {product-title-short} 4.0 and later
```

Comment:
We usually add TODOs because this makes finding unfinished things easier.
```suggestion
* TODO: add link to Upgrading from {product-title-short} 3.74 and earlier to {product-title-short} 4.0 and later
```
Up to you :-)